app/src/main/res/values/styles.xml

```diff
@@ -16,16 +16,20 @@
     </style>
 
     <style name="Hedvig.Theme.KeyGear.ItemDetail">
-        <item name="colorControlNormal">@color/white</item>
+        <item name="colorControlNormal">?colorOnPrimary</item>
```

Comment:
Does this work with the toolbar in the Key Gear Item Detail-screen? I believe the choice of `white` was intentional here 😬

Reply:
I dont see any difference but i changed it back to white

Reply:
`?colorOnPrimary` differs between light and dark theme while `@color/white` stays the same. Try it out in both themes to see the difference

Reply:
Yea i tried with white, black and pink. I didnt see any difference but I dont know what framework controls is so didnt know where to look

Reply:
It's used for the toolbar actions, iirc